examples/src/main/scala/org/apache/spark/examples/graphx/Analytics.scala

```diff
@@ -145,9 +145,9 @@ object Analytics extends Logging {
           // TriangleCount requires the graph to be partitioned
           .partitionBy(partitionStrategy.getOrElse(RandomVertexCut)).cache()
         val triangles = TriangleCount.run(graph)
-        println("Triangles: " + triangles.vertices.map {
+        println(s"Triangles: ${triangles.vertices.map {
```

Comment:
This is probably more readable if the expression is stored in a local val first. Anything nontrivial like this gets hard to parse in an interpolated string. While we're here, worth fixing up.

Reply:
sure